layouts/_default/baseof.html

```diff
@@ -46,7 +46,7 @@
         <div id="nav-header-bg"></div>
         <div id="header-main" class="container">
             <a href="{{ "/" | absURL }}" title="{{ T "logo-a-title" . }}" style="position: relative;">
-                <img id="logo" src="{{ T "logo" . | absURL }}" title="{{ T "logo-title" . }}">
+                <img class="logo" src="{{ T "logo" . | absURL }}" title="{{ T "logo-title" . }}">
```

Comment:
Thanks for the PR. There is really only supposed to be one logo per page. The 404 page is a temporary exception that will soon not be relevant anymore. Thus, I would rather not change this from an `id` to a `class` as it makes less sense semantically.

If you think that it is necessary that we fix this now, please just allow both cases in the CSS and keep this file as-is.